sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -126,6 +127,16 @@ def key_id(self):
         """
         return self._key_id.source_id
 
+    @classmethod
+    def from_jwk(cls, jwk):
+        # type: (dict) -> CryptographyClient
+        """Creates a client that can only perform cryptographic operations locally.
+
+        :param dict jwk: the key's cryptographic material, as a dictionary.
+        """
+        key_id = "https://key-vault.vault.azure.net/keys/local-key"
```

Comment:
Using a real URL as a signal value makes me nervous. Also, I think we should try to use "kid" from the JWK, if it has one. That entails relaxing KeyProperties to return None for the properties it would normally parse from a Key Vault identifier, which seems okay to me, and doing so would also allow defaulting to None here 🤔

Reply:
That's fair, I'd prefer not to use a valid URL either. But there's then also the question of how we initialize the `KeyVaultClientBase` without a vault URL 🤔